chapters/hp-exam.tex

```diff
@@ -3,8 +3,7 @@ \chapter{Final Exam}
 \section{The following was posted at the end of Chapter 113:}
 
 {%
-\setlength{\parindent}{0pt}
-\setlength{\parskip}{.5\baselineskip}
+\setlength{\parindent}{0pt}\setlength{\parskip}{.5\baselineskip}
```

Comment:
Does this really need to be reformatted?

Reply:
no, just my preference, will revert it, but still remove the other duplicated one, as that lead to an invalid file.